Feature/festival register page

MOERK-code opened the pull request.

MOERK-code committed:
fixes a bug in the check for save festival and code refractoring

MOERK-code committed:
Merge dev into feature/festival-register-page

MOERK-code committed:
Refractored FestivalRegister.cs

robinfredriks reviewed (commented):
Documentation could use some work, XML tags shouldn't  be empty or non-descriptive

Samsabal committed:
Refractored some code in FestivalRegister

Made a switch case of all else ifs and removed ResetFestivalAndTicketRegistration.

Samsabal committed:
Moved FestivalRegister into a folder.

Samsabal committed:
Refractored Festival Register plz send help

MOERK-code committed:
Removed unnecessary code and changed FestivalHandler.cs

Samsabal committed:
Update Modifier.cs

Samsabal committed:
Update Modifier.cs

Samsabal reviewed (approved):
Helemaal mooi!

Samsabal committed:
Merge branch 'dev' into feature/festival-register-page
